Ginger/Ginger/Actions/ApiActionsConversion/ApiActionsConversionIntro.md

```diff
@@ -0,0 +1,5 @@
+﻿### 1- Why to convert web services actions to API Models?
+Web services Actions (from type WebAPI RESR/SOAP) should be converted to API Models in order to enjoy the benefits of using API Models like improve Solution maintenance by placing the API configurations on repository level and separate it from the execution flow on which only the data is been set.
+
+### 2- What will happen during the conversion process?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4464228)